src/main/resources/run-class.sh

```diff
@@ -98,9 +98,17 @@ if [ -e conf/env.conf ]; then
   CONF_FILE="-Dconfig.file=conf/env.conf"
 fi
 
+#Add a cache-key config, user tests for existence
+ADD_CACHE_KEY=""
+if [ -e conf/cacheKey.conf ]; then
+  CACHEKEY_FILE_CONTENTS = $(<cacheKey.conf)
+  CACHEKEY=${CACHEKEY_FILE_CONTENTS##*: }
```

Comment:
If you're just catting to an environment variable, you should fix the file contents to be the SHA value directly, so that you don't need janky bash regex magic.